src/main/resources/resources/logisim/strings/gui/gui_de.properties

```diff
@@ -27,64 +27,64 @@ showStateDialogTitle = Zustand für %s
 ButtonExport = Exportieren...
 ButtonLoad = Datei laden
 ChronoTitle = Chronogramm
-# ==> ChronoHelp =
-# ==> ChronoPrintTitle =
-# ==> ButtonExportAsImage =
+ChronoHelp = Zeigt Signale in Echtzeit an.
+ChronoPrintTitle = Impulsdiagramm: %s von %s
+ButtonExportAsImage = Als Bild exportieren
 InputFileLoaded = Geladene Datei:
 InputFileNoSysclk = Kein Sysclk-Signal gefunden
 SimStatusCurrentScheme = Simulation des aktuellen Schemas
 SimStatusName = Status:
 SimStatusNoSignal = Kein Signal im Simulator->Chronogramm ausgewählt
 SimStatusNoSysclk = Hinzufügen des Signals 'sysclk' im Simulatoren->Chronogramm für Echtzeitbetrieb oder Laden der Datei
-# ==> NoSignalsSelected =
-# ==> optionsTab =
-# ==> optionsHelp =
-# ==> selectionLabel =
-# ==> modeLabel =
-# ==> timingLabel =
-# ==> historyLabel =
-# ==> historyUnlimited =
-# ==> historyLimit =
-# ==> timeScale =
-# ==> gateDelay =
-# ==> cycleLength =
-# ==> nsFormat =
-# ==> usFormat =
-# ==> msFormat =
-# ==> sFormat =
-# ==> perSecond =
-# ==> perTick =
-# ==> stepTime =
-# ==> stepCoarse =
-# ==> stepCoarseDescription =
-# ==> fineDetail =
-# ==> stepFineDescription =
-# ==> realTime =
-# ==> realCoarse =
-# ==> realCoarseDescription =
-# ==> realFine =
-# ==> realFineDescription =
-# ==> clockTime =
-# ==> clockCoarse =
-# ==> clockCoarseDescriptionDual =
-# ==> clockCoarseDescriptionRising =
-# ==> clockCoarseDescriptionFalling =
-# ==> clockCoarseDescriptionHigh =
-# ==> clockCoarseDescriptionLow =
-# ==> clockFine =
-# ==> clockFineDescription =
-# ==> clockSourceLabel =
-# ==> clockSourceNone =
-# ==> selectClockDriverMessage =
-# ==> selectClockObserverMessage =
-# ==> selectClockMissingMessage =
-# ==> selectClockMultipleMessage =
-# ==> clockDisciplineLabel =
-# ==> clockDisciplineRising =
-# ==> clockDisciplineFalling =
-# ==> clockDisciplineDual =
-# ==> clockDisciplineHigh =
-# ==> clockDisciplineLow =
+NoSignalsSelected = Momentan sind keine Signale ausgewählt. | Füge der Schaltung Pins hinzu, oder wähle ein Signal im Einstellungsfenster.
+optionsTab = Optionen
+optionsHelp = Signale, Betriebsmodus und Geschwindigkeit des Impulsdiagramms einstellen
+selectionLabel = Signale und Formate
+modeLabel = Aufzeichen-Modus
+timingLabel = Timing-Optionen
+historyLabel = Verlauf
+historyUnlimited = Endloser Verlauf speichern
+historyLimit = Anzahl zu speichernden Werte:
+timeScale = Zeitachse:
+gateDelay = Gatter Laufzeit:
+cycleLength = Taktcyclus: %s Ticks
```

Comment:
"Taktcyklus" -> "Taktzyklus"

Instead of "Taktzyklus", you may consider "Taktperiode", "Periodendauer", or "Taktlänge".